.vscode/settings.json

```diff
@@ -0,0 +1,18 @@
+{
+    "cSpell.words": [
+        "behaviour",
+        "Colour",
+        "customised",
+        "Deathmatch",
+        "gamemodes",
+        "gameplay",
+        "headshot",
+        "hitbox",
+        "loadout",
+        "respawn",
+        "respawning",
+        "respawns",
+        "unconnectable",
+        "Unported"
+    ]
+}
```

Comment:
This file is still present in the latest revision.

Reply:
I have removed the file